lib/orbital/ext/active_merchant/active_merchant.rb

```diff
@@ -5,6 +5,17 @@ module Billing
 
     class OrbitalGateway
 
+      API_VERSION = '7.0.1'
+
+      POST_HEADERS = {
+          "MIME-Version" => "1.1",
+          "Content-Type" => "application/PTI70",
```

Comment:
It looks like that's the only change? If so, how about doing it in the `merge` below (see `commit`), since we already override some of them?